Big-Sur.md

```diff
@@ -1,60 +1,40 @@
-### Big Sur - Rough Notes
+# Big Sur Installation
 
-#### Note: Installation (for the most part) remains the same, except for the retrieval of the DMG file.
+Note: Installation (for the most part) remains the same, except for the retrieval of the DMG file.
 
-- Fetch the `Big Sur` installer using the following command (as usual):
+# Install Dependencies
 
-  ```
-  ./fetch-macOS.py
-  ```
-
-- Unpack the download InstallAssistant.pkg file:
-
-  ```
-  sudo apt install p7zip-full -y  # install the unpacking tool
-  ```
-
-  ```
-  7z e -txar InstallAssistant.pkg '*.dmg'  # extract files
-  ```
-
-  At the end of this step, we get the `SharedSupport.dmg` file.
-
-- Extract `BaseSystem.dmg` from this `SharedSupport.dmg` file:
-
-  ```
-  7z e -tdmg SharedSupport.dmg 5.hfs  # extract support dmg
-
-  mkdir ~/stuff
-  sudo mount -oloop *.hfs ~/stuff  # mount the hfs filesystem
-
-  7z l ~/stuff/com_apple_MobileAsset_MacSoftwareUpdate/*.zip  # list files from this zip file
-  ...
-  2020-11-06 18:57:48 .....    652236311    646767350  AssetData/Restore/BaseSystem.md
-  ```
-
-  There is the required `BaseSystem.dmg` file. To unzip it, first make sure you
-  are in the base directory for `OSX-KVM` and then extract the `BaseSystem.dmg`
-  file.
-
-  ```
-  cd ~/OSX-KVM/
-
-  7z e ~/stuff/*MacSoftwareUpdate/*.zip AssetData/Restore/Base*.dmg
-
-  sudo umount ~/stuff
-  ```
+## Fedora
+`sudo dnf install p7zip p7zip-plugins qemu git wget libguestfs-tools`
 
-* Convert this extracted `BaseSystem.dmg` file into the `BaseSystem.img` file.
+Download and install: https://src.fedoraproject.org/rpms/uml_utilities
 
-  ```
-  qemu-img convert BaseSystem.dmg -O raw BaseSystem.img
-  ```
+## Ubuntu
+`sudo apt-get install qemu uml-utilities virt-manager git wget libguestfs-tools p7zip-full -y`
 
-- Follow the [main documentation](README.md#installation-preparation) as this point.
+# Creating Installation Media and Disk Image
+Open terminal and copy and paste the following code blocks below each step.
```

Comment:
Do we really want to be so explicit in our documentation (Open Terminal... Copy-Paste following things)?

I prefer not to.

Reply:
I think having some bash script to automate everything (similar to your python fetch script) is ideal, where the user only needs to specify the OS version and disk image size. I added the "copy and paste instruction" at the last minute, but I think the dependencies coming first makes for better flow.

repo cloned -> running scripts that will generate files inside repo -> take the file(s) you need / initialize VM.

If it can be generalized into these 3 steps, I think the folder for mounting the HFS should also be in the repo so you're not creating a "stuff" folder in home.

Reply:
I have been thinking about writing an ansible playbook to configure the (local)host for a while now. It could be a good alternative to a bash script in terms of the expressiveness offered.

Ack on the `stuff` folder thing. Fixing it now.

Ack on the overall flow too. The file has grown organically over time and hasn't been reviewed as a single-whole-entity for a while. Let me read it again.